src/composables/latestTransactionList.ts

```diff
@@ -98,6 +121,16 @@ export function useLatestTransactionList({ store }: IDefaultComposableOptions) {
     { deep: true },
   );
 
+  watch(
+    localPendingTransactionsList,
+    async (transactions) => {
+      if (transactions.length) {
+        await updateTransactionListData(transactions);
+      }
+    },
+    { deep: true, immediate: true },
+  );
```

Comment:
Do you think we should reload all of the transactions if there's one pending? What do you think about just pushing that one on top of the list?

Reply:
Push could be resolved before fetching the list. We should have always the newest data, when we want "push" something by myself.